qubespolicy/__init__.py

```diff
@@ -121,11 +125,11 @@ def __init__(self, line, filename=None, lineno=None):
         self.filename = filename
 
         try:
-            self.source, self.target, self.full_action = line.split()
+            self.source, self.target, self.full_action = line.split(maxsplit=2)
         except ValueError:
             raise PolicySyntaxError(filename, lineno, 'wrong number of fields')
 
-        (action, *params) = self.full_action.split(',')
+        (action, *params) = self.full_action.replace(' ', '').split(',')
```

Comment:
IIUC, this means that params can be additionally separated by spaces preceding or following the comma, but comma is still mandatory and tabs won't work (which is inconsistent to separators between source, target and full_action, where they are perfectly fine).